content/updates/2020-07/1-easily-fix-errors-on-your-connections.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/fix-errors-on-your-connections.png
+pim_announcement_alt_img: "Enriched error message"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+We improved the connection error messages by adding information that will help you to fix them. In the `Monitoring` page, you will now find links to other PIM page or help center pages, and some data about the related product.  
```

Comment:
```suggestion
We improved the connection error messages by adding information that will help you to fix them. In the `Monitoring` page, you will now find links to other PIM pages or help center pages, and some data about the related product.  
```

Reply:
What do you mean by "related product"?

Reply:
We only monitor product errors so there is always a product related to an error message